skxray/core.py

```diff
@@ -729,6 +730,118 @@ def wedge_integration(src_data, center, theta_start,
     raise NotImplementedError()
 
 
+def radial_integration(image_array, calibrated_center,
```

Comment:
I am torn on this API.  On one hand, this is convenient, on the other hand I would like to have this function return bin-edges in pixels and be converted later by what ever code calls this function.